dev-util/debugedit/files/debugedit-5.0-musl-error.h-fix.patch

```diff
@@ -0,0 +1,1435 @@
+# musl doesn't provide error.h as a result debugedit is failing to build on
```

Comment:
I don't think the patch is bad, but bleh, it's sooooo big.

@blueness?

Reply:
(It might be I upload it to devspace, just wondering if we can do something better here.)

Reply:
@thesamesam @blueness Is the current implementation alright?

Reply:
looks good